Sources/ConsulServiceDiscovery/Consul.swift

```diff
@@ -160,7 +159,7 @@ public final class Consul: Sendable {
                         var buffer = buffer
                         let bytes = buffer.readBytes(length: buffer.readableBytes)
                         if let bytes {
-                            let dict = try XJSONDecoder().decode([String: [String]].self, from: bytes)
+                            let dict = try JSONDecoder().decode([String: [String]].self, from: Data(bytes))
```

Comment:
So we will have 2 memory allocations: the buffer returned by `ByteBuffer.readBytes()`, and the second one with instantiating the `Data`.
We could avoid one allocation if would read data directly from the buffer, like that:
```
                        var buffer = buffer
                        try buffer.withUnsafeReadableBytes { bytes in
                            let dict = try JSONDecoder().decode([String: [String]].self, from: Data(bytes))
                            promise.succeed(Array(dict.keys))
                        }
```